src/utils/migration/02_registration.ts

```diff
@@ -93,22 +93,22 @@ const main = async () => {
   // https://docs.safe.global/advanced/smart-account-supported-networks?service=Transaction+Service&service=Safe%7BCore%7D+SDK
 
   // Setup the SafeKit
-  const safeKit = await SafeKit.init(config);
+  // const safeKit = await SafeKit.init(config);
 
-  // If admin given is not a Safe owner, fail early
-  if (!await safeKit.isOwner(migrationAdmin.address)) {
-    throw new Error(
-      `Migration admin ${migrationAdmin.address} is not a Safe owner. Ensure the admin address is added as a Safe owner`
-    );
-  }
+  // // If admin given is not a Safe owner, fail early
+  // if (!await safeKit.isOwner(migrationAdmin.address)) {
```

Comment:
Uncomment this. If you want some changes for local testing add to check the `hre.network.name === "hardhat"`